src/Agent/NewRelic/Agent/Core/WireModels/LogEventWireModel.cs

```diff
@@ -70,5 +86,30 @@ public LogEventWireModel(long unixTimestampMS, string message, string level, str
             TraceId = traceId;
             Priority = priority;
         }
+
+        public LogEventWireModel(long unixTimestampMS, string message, string level, ICollection<string> errorStack, string errorMessage, string errorClass, string spanId, string traceId)
+        {
+            TimeStamp = unixTimestampMS;
+            Message = message.TruncateUnicodeStringByBytes(MaxMessageLengthInBytes);
+            Level = level;
+            SpanId = spanId;
+            TraceId = traceId;
+            ErrorStack = string.Join(" \n", errorStack);
```

Comment:
Since this will be going out as JSON, does the newline need to be escaped?

Reply:
I have not seen any errors in testing, the data comes in properly.